server/common/comments/get.ts

```diff
@@ -0,0 +1,39 @@
+import { NextApiRequest } from "next";
+import { WhereOptions } from "sequelize";
+
+import models from "db/models";
+
+import { isGovernorSigned } from "helpers/handleIsGovernor";
+
+export default async function get(req: NextApiRequest) {
+  const { type, id } = req.query;
+
+  const isGovernor = await isGovernorSigned(req.headers);
+
+  const filters: WhereOptions = {};
+
+  if(!isGovernor) filters.hidden = false;
```

Comment:
I made it based on the story in jira. We don't need it because both are unique. There is no way to have a proposalId with the same Id in another issue.